namespace.go

```diff
@@ -139,9 +139,12 @@ func NamespaceWithWebSocket() NamespaceOption {
 // The Azure Environment used can be specified using the name of the Azure Environment set in "AZURE_ENVIRONMENT" var.
 func NamespaceWithEnvironmentBinding(name string) NamespaceOption {
 	return func(ns *Namespace) error {
+		if ns.ResourceURI == "" {
+			ns.ResourceURI = serviceBusResourceURI
+		}
 		provider, err := aad.NewJWTProvider(
```

Comment:
Looking at the impl for this, it also uses a hard-coded ResourceURI for event hubs which I'm guessing also needs to be fixed.

Reply:
right, 
```
func NewJWTProvider(opts ...JWTProviderOption) (*TokenProvider, error) {
	config := &TokenProviderConfiguration{
		ResourceURI: eventhubResourceURI,
	}
...
```
so we still need to call aad.JWTProviderWithResourceURI(ns.getResourceURI()) to work around their bug.
we can remove it once the upstream lib is not hardcoding it.